tuna/utils/db_utility.py

```diff
@@ -46,6 +55,24 @@ def get_id_solvers():
   return id_solver_map_c, id_solver_map_h
 
 
+def session_retry(session, callback, actuator, logger=LOGGER):
```

Comment:
I believe there are other places in our code, where we can make use of this function and do some cleanup, perhaps as a follow up PR

Reply:
@reidkwja Can you please follow up on the comment above ?

Reply:
@JehandadKhan I was thinking the same thing. But I'm waiting for 744, because of change conflicts there.